### DISCUSSION TITLE
Welcome to OpenDJ Discussions!

### DISCUSSION THREAD

**Discussion by maximthomas** [OpenIdentityPlatform/OpenDJ#158] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petergietz** [comment]
Hi all, this is Peter, CEO of a German Open Source IAM company and co-organizer of LDAPCon, and to be honest, for now I only want to make you aware of the the next LDAPCon (https://ldapcon.org/2026/), which will take place in Tübingen, Germany on October 6-8, 2026. The call for papers is open until July 17th. May be someone here has something interesting to present. Please excuse this plug.
